SocialLogin.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>

```

Comment:
- Indentation
- Extra lines
- Space between selector and `{`
- Space between property and value
- New line between selectors